src/renderer/services/state/index.js

```diff
@@ -1,5 +1,25 @@
 import { send } from 'services/ipc';
-import { SET_KEY } from 'common/events';
+import {
+  IS_INITIALIZED,
+  GET_STORE_FILE_PATH,
+  SET_KEY,
+  LOAD_STATE,
+} from 'common/events';
+
+/**
+ * Asks main process if application was initialized.
+ * Meaning, is the user locally registered ?
+ *
+ * @returns {Promise<bool>} - Is the app initialized ?
+ */
+export const isInitialized = () => send(IS_INITIALIZED);
+
+/**
+ * Gets the store file path used by developer-vault.
+ *
+ * @returns {Promise<string>} - The store file path.
```

Comment:
I think we should just specify the 'string' type in `@returns`, and setting `@async` to know that this is a Promise.